### DISCUSSION TITLE
Its not Laravel 10 compatible

### DISCUSSION THREAD

**Discussion by JonathanVorich** [thedevdojo/voyager#5739] [General]
I really like to use the tcg/voyager package. But its not compatible with Laravel 10.
Can you please release a version for Laravel 10?

If i try to install in my new Project it returns the following error: 

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires tcg/voyager ^1.6 -> satisfiable by tcg/voyager[v1.6.0].
    - tcg/voyager v1.6.0 requires illuminate/support ~8.0|~9.0 -> found illuminate/support[v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.4] but these were not loaded, likely because it conflicts with another require.
```

**Comment by DijDij62** [comment]
Same issue, not compatible with laravel 10

**Reply by justgkp** [reply]
Hope the team will update  it soon to make it Laravel 10 compatible...waiting eagerly!

**Comment by jomisa** [comment]
Laravel 10 Support please!!!! 😭

**Comment by silicahd** [comment]
Based on the fact that 40% of the links on the homepage no longer work I am inclined to bet that this project is fairly dead. I wish they would have added a plugin store much like filament and charge a fees for every sale. This way the system would still be profitable and up. But no such thing ever came to be.
